### ISSUE TITLE
Migration database 0.99 to 1.2 with LDAP

### ISSUE THREAD

**Issue by hger** [indico/indico#1665]
I get this error when runing the migrate command:
(python /opt/indico/bin/migration/migrate.py --prev-version 0.99)

```
# Convert all LDAP identities to lowercase (1.2)
  Processing 1/193 
Migration failed! DB may be in  an inconsistent state:
Traceback (most recent call last):
  File "/opt/indico/bin/migration/migrate.py", line 1086, in main
    dry_run=args.dry_run)
  File "/opt/indico/bin/migration/migrate.py", line 1032, in runMigration
    task(dbi, withRBDB, prevVersion)
  File "/opt/indico/bin/migration/migrate.py", line 947, in lowercaseLDAPIdentities
    assert identity.getId() not in idx
AssertionError
```

**Comment by ThiefMaster** [comment]
fix is in this gist at the moment (not committed yet): https://gist.github.com/ThiefMaster/a7e8c54cb6481147d299

**Comment by pferreir** [comment]
Merged in a580055541072d337c7694e5c9b5a33fa059d586
